crates/filament/src/resolver.rs

```diff
@@ -32,6 +32,16 @@ impl From<&cmdline::Opts> for Resolver {
 }
 
 impl Resolver {
+    pub fn new(lib: Vec<PathBuf>, input: PathBuf) -> Self {
```

Comment:
What's the use case for this method?

Reply:
I'm still making changes, so this is a draft PR and not ready for review. There's a lot of broken stuff right now.

Reply:
I want to make a resolver without constructing an instance of the filament command line options.